DataProcessing/dataHandler.py

```diff
@@ -45,28 +49,41 @@ def save_crop(self, datapath):
             mmcv.imwrite(face_to_write, os.path.join(datapath, f'Face_{self.unique_crop_name}.png'))
 
     def check_if_face_img(self):
-        return self.face_img is not None and self.face_img is not self.face_img.numel()
+        self.is_face = self.face_img is not None and self.face_img is not self.face_img.numel()
+        return self.is_face
```

Comment:
As it is a class attribute, we don't need to return it here. We can just check if self.is_face later.

Reply:
I doubted this for a bit. I think it is better to keep as is, as the function updates the state of is_face, and also expects a return value. We could refactor to call the update and then query the attribute but I believe it is redundent

Reply:
Looked at the usage of the function and now I understand what you mean.
I'm good with leaving it this way